torch/_ops.py

```diff
@@ -791,6 +791,11 @@ def _handle(self):
             )
         return self._lazy_handle
 
+    def _unsafe_set_tags(self, tags: List[torch._C.Tag]):
+        return torch._C._unsafe_set_tags(
+            tags, self._schema.name, self._schema.overload_name
+        )
```

Comment:
Did you forget to check in the python bindings for these? I don't see them in this PR

Reply:
yes, added